src/darwin/Framework/CHIP/CHIPSetupPayload.h

```diff
@@ -28,6 +28,13 @@ typedef NS_ENUM(NSUInteger, CHIPRendezvousInformationFlags) {
     kRendezvousInformationAllMask = kRendezvousInformationSoftAP | kRendezvousInformationBLE | kRendezvousInformationOnNetwork,
 };
 
+typedef NS_ENUM(NSUInteger, CHIPCommissioningFlow) {
+    kCommissioningFlowStandard = 0, // Device automatically enters pairing mode upon power-up
+    kCommissioningFlowUserActionRequired = 1, // Device requires a user interaction to enter pairing mode
+    kCommissioningFlowCustom = 2, // Commissioning steps should be retrieved from the distributed compliance ledger
+    kCommissioningFlowInvalid = 3,
```

Comment:
Does the spec mention invalid enum, I thought it was just a reserved bit. 
I'd expect the code to have 3 enums and if we ever get value 3, we'd return an error. Or the adopter would know this is invalid as we did not define it.

Reply:
It is a reserved value, but here I need something to represent it while converting to the Darwin type. Normally it shouldn't happen.